lib/render/ParticleRenderer.cpp

```diff
@@ -25,6 +25,7 @@
 #include <vapor/glutil.h>    // Must be included first!!!
 
 #include <vapor/Renderer.h>
+#include <vapor/Particle.h>
```

Comment:
Isn't it the case that `flow::Particle` is no longer used? If so, why include it here?

Reply:
Thanks, removed.